lib/cppcheck.cpp

```diff
@@ -745,8 +745,9 @@ unsigned int CppCheck::checkFile(const std::string& filename, const std::string
 
         // Get configurations..
         if ((mSettings.checkAllConfigurations && mSettings.userDefines.empty()) || mSettings.force) {
-            Timer t("Preprocessor::getConfigs", mSettings.showtime, &s_timerResults);
```

Comment:
In this case I think the RAII is preferable

Reply:
I did it out of consistency.